src/cmds/Makefile.am

```diff
@@ -194,7 +194,7 @@ qmgr_CPPFLAGS = \
 qmgr_LDADD = \
 	${common_libs} \
 	@editline_lib@
-qmgr_SOURCES = qmgr.c ${common_sources}
+qmgr_SOURCES = qmgr.c qmgr_sup.c ${common_sources}
```

Comment:
is this change intentional?

Reply:
Hi @hirenvadalia yes. We are bifurcating some part of code in qmgr, to make it platform independent.